app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,10 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @like = @current_user.likes.new(post_id: @post.id)
+
+    respond_to do |format|
+      format.html { redirect_to @post }
+    end
+  end
```

Comment:
The provided routes.rb file already includes the required route for creating a like. The route POST /posts/:post_id/likes(.:format) likes#create is present in the file:

ruby

post 'like', on: :member

This route is nested within the resources :posts block and will create a like for a specific post. The on: :member option indicates that the route should be scoped to an individual member (post) rather than the collection.

Therefore, there is no need to add any additional routes for creating likes. The existing route should handle the functionality correctly.

If you have any further questions or need clarification, please let me know.